test/parallel/test-querystring.js

```diff
@@ -232,8 +232,20 @@ assert.doesNotThrow(function() {
   assert.strictEqual(f, 'a:b;q:x%3Ay%3By%3Az');
 }
 
+// empty string
+assert.strictEqual('', qs.stringify());
```

Comment:
Nit: can you please swap the order of the parameters? `actual` first, then `expected`.